source/overview/license-and-subscription.rst

```diff
@@ -71,9 +71,26 @@ You may incur retroactive charges for any unique users added that exceed the lic
 Renewing an Enterprise license
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ability to renew your license in the Customer Portal will be available in a future release.
+You can renew your self-managed Mattermost Enterprise license with a credit card if you have a standard Mattermost contract. When you renew your license, you can also increase the number of active users.
 
-To renew your license in the meantime, contact your Customer Success Manager to start the renewal process. Alternatively, `contact us <https://mattermost.com/contact-us/>`_ for assistance.
+If you're a Mattermost Cloud customer, your subscription auto-renews so you don't need to follow this process.
```

Comment:
Out of interest, how do Mattermost Cloud customers change the parameters of their subscription? We mention it for self-hosted and should mention it for Mattermost Cloud for consistency.

Reply:
The parameters (e.g. users) are automatically updated for them.